super_editor/test/super_editor/supereditor_keyboard_test.dart

```
@@ -281,6 +282,34 @@ void main() {
       });
     });
   });
+
+  group('SuperEditor software keyboard', () {
+    testWidgetsOnIos('pressing tab indent list', (tester) async {
+      final testContext = await _pumpUnorderedList(tester);
+      final doc = testContext.editContext.editor.document;
+      final nodeId = doc.nodes.first.id;
+
+      // Ensure we started with indentation level 0.
+      expect((doc.nodes.first as ListItemNode).indent, 0);
```

How about we add a method to the inspector called `DocumentNode getNodeAt<T>(int index)`, which returns the node at the given index, cast to the given type. It should throw an exception for a bad index, and it should throw an exception for a bad type.

Since you access the node twice, and you use the node ID twice, you should probably store the node in a local variable and use the node across those sitautions.

---

Method added.